client/package.json

```diff
@@ -39,7 +40,8 @@
     "@storybook/addon-links": "^5.3.18",
     "@storybook/addons": "^5.3.18",
     "@storybook/preset-create-react-app": "^2.1.1",
-    "@storybook/react": "^5.3.18"
+    "@storybook/react": "^5.3.18",
+    "storybook-addon-themes": "^5.4.1"
```

Comment:
not sure what you installed here. I don't think you need to install an add-on for storybook theming. at least I can't find anything online. You just need an add-on for emotion-theming which you already installed as I can see. 
Maybe check this file/add-on again

Reply:
yeah I'm not sure either where this came from and I think this is the reason for the whole thing not running :/ I think I found the right addon right now.